tests/test_nested_items.py

```diff
@@ -0,0 +1,57 @@
+import unittest
+
+from itemloaders import ItemLoader
+
+
+class NestedItemTest(unittest.TestCase):
+    """Test that adding items as values works as expected."""
+
+    def test_attrs(self):
+        try:
+            import attr
+        except ImportError:
+            self.skipTest("Cannot import attr")
+
+        @attr.s
+        class TestItem:
+            foo = attr.ib()
+
+        item = TestItem(foo='bar')
+        il = ItemLoader()
+        il.add_value('item_list', item)
+        self.assertEqual(il.load_item(), {'item_list': [item]})
+
+    def test_dataclass(self):
+        try:
+            from dataclasses import dataclass
+        except ImportError:
+            self.skipTest("Cannot import dataclasses.dataclass")
+
+        @dataclass
+        class TestItem:
+            foo: str
+
+        item = TestItem(foo='bar')
+        il = ItemLoader()
```

Comment:
I just wonder if it makes sense to move this common piece of code to a function and call it in the tests, with the proper item argument